src/main/java/com/aws/greengrass/deployment/DefaultDeploymentTask.java

```diff
@@ -188,6 +189,7 @@ public DeploymentResult call() throws InterruptedException {
             // skip root packages if device does not belong to that group anymore
             if (!groupTopics.getName().equals(deploymentDocument.getGroupName())
                     && (groupTopics.getName().startsWith(DEVICE_DEPLOYMENT_GROUP_NAME_PREFIX)
+                    || groupTopics.getName().equals(LOCAL_DEPLOYMENT_GROUP_NAME)
```

Comment:
Should this be `THING_GROUP_RESOURCE_NAME_PREFIX + LOCAL_DEPLOYMENT_GROUP_NAME`?

Reply:
No, LOCAL_DEPLOYMENT_GROUP_NAME is used when group name is not specified for a local deployment. It is not a valid thing group and hence does not have the prefix.